src/pages/Xvs/constants.ts

```diff
@@ -0,0 +1,6 @@
+import BigNumber from 'bignumber.js';
+import { getToken } from 'utilities';
+
+export const XVS_DECIMALS = getToken('xvs').decimals;
+export const MINTED_XVS = '23700000';
```

Comment:
I think this constant should be made global to the app, in case other pages need it too since this is true for the entire app.

Reply:
Got you 👍